src/server/game/World/World.cpp

```diff
@@ -1123,7 +1123,18 @@ void World::LoadConfigSettings(bool reload)
         sLog->outError("Battleground.ReportAFK (%d) must be <10. Using 3 instead.", m_int_configs[CONFIG_BATTLEGROUND_REPORT_AFK]);
         m_int_configs[CONFIG_BATTLEGROUND_REPORT_AFK] = 3;
     }
-
+    m_int_configs[CONFIG_BATTLEGROUND_RESPAWN_TIME]        = sConfigMgr->GetIntDefault("Battleground.RespawnTime", 30);
+    if (m_int_configs[CONFIG_BATTLEGROUND_RESPAWN_TIME] < 3)
+    {
+        sLog->outError("Battleground.RespawnTime (%i) must be in range 3..120. Using 30 instead.", m_int_configs[CONFIG_BATTLEGROUND_RESPAWN_TIME]),
+        m_int_configs[CONFIG_BATTLEGROUND_RESPAWN_TIME] = 30;
+    }
+    else if (m_int_configs[CONFIG_BATTLEGROUND_RESPAWN_TIME] > 120)
+    {
+        sLog->outError("Battleground.RespawnTime (%i) must be in range 3..120. Using 30 instead.", m_int_configs[CONFIG_BATTLEGROUND_RESPAWN_TIME]),
+        m_int_configs[CONFIG_BATTLEGROUND_RESPAWN_TIME] = 30;
+    }
```

Comment:
Can't you just do 
```cpp
if (m_int_configs[CONFIG_BATTLEGROUND_RESPAWN_TIME < 3 || m_int_configs[CONFIG_BATTLEGROUND_RESPAWN_TIME > 120)
```
?

Reply:
Looks like I can, thanks. I don't really know C++ :P

Reply:
Less clutter :D

Also, change the outError to 3 - 120 instead of ...